packages/core/content-type-builder/server/src/services/builder.ts

```diff
@@ -8,25 +8,36 @@ export const getReservedNames = () => {
       'time',
       'upload',
       'then', // https://github.com/strapi/strapi/issues/15557
-      'rest', // https://github.com/strapi/strapi/issues/13643
       'document',
     ],
     // attributes are compared with snake_case(name), so only snake_case is needed here and camelCase + UPPER_CASE matches will still be caught
     attributes: [
-      // TODO V5: these need to come from a centralized place so we don't break things accidentally in the future
+      // TODO: these need to come from a centralized place so we don't break things accidentally in the future and can share them outside the CTB, for example on Strapi bootstrap prior to schema db sync
+
+      // ID fields
       'id',
-      'documentId',
       'document_id',
+
+      // Creator fields
       'created_at',
       'updated_at',
       'published_at',
       'created_by_id',
       'updated_by_id',
-      'locale', // conflicts with internal locale field
-
-      // not actually breaking but we'll leave it to avoid confusion
+      // does not actually conflict because the fields are called *_by_id but we'll leave it to avoid confusion
       'created_by',
       'updated_by',
+
+      // Used for Strapi functionality
+      'entry_id',
+      'status',
+      'localizations',
+      'meta',
+      'locale',
+      // TODO: remove these in favor of restricting the strapi_ prefix
```

Comment:
Is there an ETA for this?

Reply:
It's in the v5 tasks, so hopefully soon!